ajax/libs/roundabout/package.json

```diff
@@ -19,5 +19,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/fredleblanc/roundabout.git"
-  }
+  },
+  "license":"BSD-3-Clause",
```

Comment:
Let's add a space between `:` and `"`.
Like from A to B.
- A
```
"license":"BSD-3-Clause",
```
- B
```
"license": "BSD-3-Clause",
```
Thank you.